install.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 Google LLC
```

Comment:
Why is it just `./install.ts`?  Should be in `system-test/`?

Reply:
Seems like you have one in `system-test/` and this one needs to be removed. Can you verify `synthtool` does not place it?

Reply:
Oh I copied paste at the wrong place at first and forgot to remove it, deleted it, thanks